### PR TITLE
EliaDev-190 bmw prequalification

### PR DESCRIPTION
**Pull request by jrhender**
Fixing up `asset-operator-server` and `prequalification-client` to prep for end user

Story https://energyweb.atlassian.net/browse/ELIADEV-190

- [x] asset-operator-server README updates
- [x] feat(Asset): add claimTypeVersion
- [x] feat(server config): add keyManager path
- [x] chore(IamClientLibFactory): update iam-client-lib to 2.2.0-alpha.5 to get claim request to work
- [x] pass in path to server config

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -74,4 +74,6 @@ keymanager.db
 # Contract ethers typechain
 ethers-types
 
-config
\ No newline at end of file
+common/config/
```
Comment (adamstaveley) [inline .gitignore:77]: This is the rush config right, wouldn't we want to commit this to vc?

**Review by adamstaveley — COMMENTED** [review]
Great docs! Looks good to me, just the question below. I will also check it locally.

File: .gitignore (hunk 1 above)
Comment (jrhender) [reply .gitignore:77]: Good question, maybe we do want it committed 🤔

Comment (jrhender) [reply .gitignore]: Removed this line so that the config files are added 👍

**Review by adamstaveley — APPROVED** [review]

**Pull request merged**